docs/In-Band Manageability Developer Guide.md

```diff
@@ -7,20 +7,20 @@
     1. [Purpose](#purpose)
     2. [Audience](#audience)
 2. [Architecture](#architecture)
-   1. [INB](#inb)
-         1. [CloudAdapter Agent](#cloudadapter-agent)
-         2. [Configuration Agent](#configuration-agent)
-         3. [Diagnostic Agent](#diagnostic-agent)
-         4. [Dispatcher Agent](#dispatcher-agent-dms)
+   1. [INB](https://github.com/intel/intel-inb-manageability/blob/Broken_link_522523/docs/In-Band%20Manageability%20Developer%20Guide.md#inbm)
```

Comment:
these links are invalid as they refer to a specific branch--need to stay with relative links